xija/model.py

```diff
@@ -168,6 +168,12 @@ def _eng_match_times(start, stop, dt):
         """
         time0 = 410270764.0
         i0 = int((DateTime(start).secs - time0) / dt) + 1
+        # Make sure i0 is always even to prevent an off-by-one
+        # issue that may result in model evolution being different
+        # by ~0.1s of degrees C if one begins the evolution at
+        # different times
```

Comment:
Can you include a link back to this PR with all the excellent plots and discussion.